setup.py

```diff
@@ -124,12 +124,12 @@ def finalize_options(self):
                       'doc': ["Sphinx == 1.2.2", "sphinx-bootstrap-theme"]},
       classifiers=classifiers,
       package_data={
+          'skbio.diversity.alpha.tests': ['data/qiime-191-tt/*'],
+          'skbio.diversity.beta.tests': ['data/qiime-191-tt/*'],
           'skbio.io.tests': ['data/*'],
           'skbio.io.format.tests': ['data/*'],
           'skbio.stats.tests': ['data/*'],
           'skbio.stats.distance.tests': ['data/*'],
-          'skbio.diversity.alpha.tests': ['data/*'],
```

Comment:
Travis is now set up to catch this, so if a dev forgets to update `package_data` in setup.py the build will fail.

I don't think we should add an empty data directory path here because by the same logic we would need to add all possible empty data directories that could occur in scikit-bio.